jenkins/tasks/main.yaml

```diff
@@ -0,0 +1,33 @@
+- name: install wget
+  apt:
+    name: wget
+    state: present
+- name: install openjdk
+  apt:
+    name: java-1.8.0-openjdk
+    state: present
+- name:  ensure the jenkins apt repository key is installed 
+  apt_key: url=https://pkg.jenkins.io/debian-stable/jenkins.io.key state=present
+  become: yes
+- name: ensure the repository is configured
+  apt_repository: repo='deb https://pkg.jenkins.io/debian-stable binary/' state=present
+  become: yes
+- name: ensure jenkins is installed
+  apt: name=jenkins update_cache=yes
+  become: yes
+- name: ensure jenkins is running  
+  service: name=jenkins state=started
+- name: enable jenkins
+  shell: systemctl enable jenkins
+- name: sleep for 30 seconds and continue with play
+  wait_for: timeout=30
+  delegate_to: localhost
+- name: init password jenkin
+  shell: cat /var/lib/jenkins/secrets/initialAdminPassword
+  changed_when: false
+  register: result
+- name: print init password jenkins
```

Comment:
I feel once we run this playbook and access jenkins it will ask us initial password, even that initial password step can be automated?